yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -204,6 +264,7 @@ export class PeerManager {
   public getPeers(includePending = false): PeerInfo[] {
     const connected = this.libP2PNode
       .getPeers()
+      .filter(peer => !this.isPrivatePeer(peer))
```

Comment:
ah sorry it wasnt clear, it doesnt need to be included here and 287 too. Sorry i only highlighted the dialqueue, i meant the whole thing